src/storage/src/render/persist_sink.rs

```diff
@@ -100,6 +104,25 @@ pub fn render<G>(
                     continue;
                 }
 
+                // Make sure that our write handle lease does not expire!
+                //
+                // We don't write to _Consensus_ but instead only write batch
```

Comment:
This paragraph was only relevant for the (compute) `persist_sink` because it has separate writers and appenders. Here, we should just remove it. Sorry for the confusion :sweat_smile:

Reply:
lol yes of course.  I should have payed better attention to the comments I was copying 🙈

Reply:
I got you! that's what reviews are for :blush: